stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -16,5 +22,76 @@ protected String onGetNodeName(View element) {
         StringUtil.removePrefix(className, "android.view.",
         StringUtil.removePrefix(className, "android.widget."));
   }
+
+  @Override
+  protected void onCopyAttributes(View element, AttributeAccumulator attributes) {
+    String id = getIdAttribute(element);
+    if (id != null) {
+      attributes.add(ID_ATTRIBUTE_NAME, id);
+    }
+  }
+
+  // Copied from Resources.resourceHasPackage because it's marked as @hide
+  private static boolean resourceHasPackage(int resid) {
```

Comment:
I really have no idea why this check exists in `View` itself.  This looks a bit like unnecessary legacy.  Specifically, even if the package id is 0x00, `Resource#getResourcePackageName` should still work.  I'd prefer to drop it and take a crash/bad result if I'm wrong :)